developer-extension/src/panel/components/tabs/eventsTab/facetList.tsx

```diff
@@ -82,19 +122,41 @@ function FacetValue({
   facetRegistry: FacetRegistry
   excludedFacetValues: ExcludedFacetValues
   onExcludedFacetValuesChange: (newExcludedFacetValues: ExcludedFacetValues) => void
+  addFacetValues: (facet: Facet, facetValue: FacetValue) => void
+  allFacetValues: ExcludedFacetValues
+  parentList: string[]
+  onlyFacet: FacetValue | null
+  setOnlyFacet: (facet: FacetValue | null) => void
 }) {
+  addFacetValues(facet, facetValue)
+
   const isTopLevel = depth === 0
   const isSelected = !excludedFacetValues[facet.path] || !excludedFacetValues[facet.path].includes(facetValue)
+  const isOnly = onlyFacet === facetValue
   const value = (
     <Flex justify="space-between" mt={isTopLevel ? 'xs' : SPACE_BETWEEN_CHECKBOX}>
       <Checkbox
         label={facet.values?.[facetValue]?.label ?? facetValue}
-        checked={isSelected}
+        checked={isSelected && !onlyFacet}
```

Comment:
**💬 suggestion:** 

- `&& !onlyFacet` is preventing the checkbox to be selected in "Only" mode.
- Disabling the checkbox when in "only" mode to prevent from side effects.
‏
```suggestion
        checked={isSelected}
        disabled={!!onlyFacet}
```